scripts/aws/cfn/microservice.yml

```diff
@@ -54,6 +54,14 @@ Parameters:
   TokenExchangeLambdaEnvironmentJwksMapping:
     Type: String
     Description: 'Mapping tra issuer e jwks url non standard in formato json'
+
+  TokenExchangeLambdaEnvironmentAllowedAudience:
+    Type: String
+    Description: Comma separated list of audience allowed to request token
```

Comment:
Comma separated list of token audience accepted.

Per lo stesso motivo modificherei TokenExchangeLambdaEnvironmentAllowedAudience in TokenExchangeLambdaEnvironmentAcceptedAudience.

Perché mentre gli issuer identificano chi ha prodotto il token, audience è il destinatario del token